palantirnet/st-bernards
# MSSB-86: Services page -- add sidebar, button tweaks, style, etc

## JIRA Ticket(s)

- See ticket https://palantir.atlassian.net/browse/MSSB-86

## Description

Read ticket.

## Testing instructions

1. Run `drush cr && drush cim -y && drush cr` 
1. Visit `http://st-bernards.local/services` and verify the work has been done and functions correctly.
